### DISCUSSION TITLE
Homebrew Core - Prepackaged with a tap including xmrig.rb coin miner

### DISCUSSION THREAD

**Discussion by U03A9** [Homebrew/discussions#3347] [Everyday usage]
Hello all. 

I work as a Security Analyst and we have been seeing alarms set off by Homebrew due to this file being present. This is a coin miner considered as unwanted software by multiple entities, including Windows Defender. 

Is there a reason this is packaged with fresh installs of Homebrew?

**Comment by SMillerDev** [comment]
It's not actually a coin miner, it's the definition to install one. And it's in the default repository because someone submitted it.
